src/Microsoft.AspNet.Razor/Chunks/ChunkTree.cs

```diff
@@ -3,11 +3,11 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.AspNet.Razor.Generator.Compiler
+namespace Microsoft.AspNet.Razor.Chunks
 {
-    public class CodeTree
+    public class ChunkTree
```

Comment:
I debated it (was thinking same thing) but in the end decided not to do it purely to reduce churn since I couldn't come up with a strong enough reason to remove it.